semcore/d3-chart/src/component/ChartLegend/LegendItem/LegendItem.tsx

```diff
@@ -13,16 +13,21 @@ import {
   StaticShapes,
 } from './LegendItem.type';
 import resolveColorEnhance from '@semcore/utils/lib/enhances/resolveColorEnhance';
+import uniqueIDEnhancement from '@semcore/utils/lib/uniqueID';
 import { PatternSymbol } from '../../../Pattern';
 import { getChartDefaultColorName } from '../../../utils';
 
+const enhance = [resolveColorEnhance(), uniqueIDEnhancement()] as const;
 class LegendItemRoot extends Component<
-  LegendItemProps & { resolveColor: ReturnType<typeof resolveColorEnhance> }
+  LegendItemProps & { resolveColor: ReturnType<typeof resolveColorEnhance> },
```

Comment:
I think, we could remove now `& { resolveColor: ReturnType<typeof resolveColorEnhance> }`